modules/services.sh

```diff
@@ -100,10 +100,11 @@ function services {
             fi
           elif source $SERVICES/install-${service_name}.sh && install ; then
             retries=0
-            while [ "$retries" -lt 2 ];
+            while [ "$retries" -lt 5 ];
             do
               if ! docker-compose --project-directory /srv/$service_name -f /srv/${service_name}/${service_name}.yml pull ; then
-                echo "retrying pull"
+                echo "retrying pull in 6 seconds"
+                sleep 6
```

Comment:
I was thinking about encapsulating the `echo` (and `sleep` in this case) in an `if [ "$retries" -lt 4 ];` so that it's not printed after the last pull attempt (since its not actually pulling again after that last one).

Reply:
Good idea.